templates/download/risc-v/tab-5.html

```diff
@@ -1,32 +1,48 @@
-			<h2>Microchip Polarfire SoC FPGA Icicle 套件</h2>
-			<h3 class="p-heading--4">Ubuntu Server预安装镜像</h3>			
-			<p>
-				<a class="p-button--positive" href="https://cdimage.ubuntu.com/releases/{{ releases.latest.full_version }}/release/ubuntu-{{ releases.latest.full_version }}-preinstalled-server-riscv64+icicle.img.xz">下载 {{ releases.latest.full_version }}</a>
-				<a class="p-button" href="https://cdimage.ubuntu.com/releases/{{ releases.lts.full_version }}/release/ubuntu-{{ releases.lts.full_version }}-preinstalled-server-riscv64+icicle.img.xz">下载 {{ releases.lts.full_version }}</a>
-			</p>
-			<p>适用于：</p>
-			<ul class="p-list">
-				<li class="p-list__item is-ticked">具有<a href="https://github.com/polarfire-soc/hart-software-services/releases/tag/v2022.10">HSS v2022.10</a>的Microchip Polarfire SoC FPGA Icicle套件</li>
-			</ul>
-		</div>
-		<div class="col-2 u-hide--medium u-hide--small">&nbsp;</div>
-		<div class="col-4 u-hide--medium u-hide--small u-align--center">
-			{{ image (
-				url="https://assets.ubuntu.com/v1/40eacb41-Microchip+PolarFire.png",
-				alt="",
-				width="800",
-				height="414",
-				hi_def=True,
-				loading="auto"
-				) | safe
-			  }}
-		</div>
-	</div>
-	<div class="u-fixed-width">
-		<h4>第一次在Microchip Polarfire SoC FPGA Icicle开发套件上安装Ubuntu？</h4>
-		<p>请阅读<a href="https://wiki.ubuntu.com/RISC-V/PolarFire%20SoC%20FPGA%20Icicle%20Kit"> 如何在Microchip Polarfire SoC FPGA Icicle开发套件上安装 Ubuntu </a>以获得更多信息。</p>
-	</div>
-</div>
\ No newline at end of file
+<div tabindex="0"
+     role="tabpanel"
+     id="quemu-sifive-image-tab"
+     aria-labelledby="quemu-sifive-image">
+    <div class="row u-vertically-center u-no-padding">
+        <div class="col-4">
+            <h2>SiFive Unmatched</h2>
+            <ul class="p-list--divided">
+                <li class="p-list__item">
+                    <h3 class="p-heading--4">Ubuntu Server预安装镜像</h3>
+                    <div class="p-notification--information is-inline">
+                        <div class="p-notification__content">
+                            <h4 class="p-notification__title">注意：</h4>
+                            <p class="p-notification__message">如果您拥有一块NVMe驱动器，我们建议您使用 Ubuntu Server 在线安装程序。</p>
+                        </div>
+                    </div>
+                    <p>
+                        <a class="p-button--positive"
+                           href="https://cdimage.ubuntu.com/releases/{{ releases.lts.full_version }}/release/ubuntu-{{ releases.lts.full_version }}-preinstalled-server-riscv64+unmatched.img.xz">下载 {{ releases.lts.full_version }}</a>
+                    </p>
+                </li>
+                <li class="p-list__item">
+                    <h3 class="p-heading--4">Ubuntu Server在线安装程序</h3>
+                    <p>
+                        <a class="p-button--positive"
+                           href="https://cdimage.ubuntu.com/releases/24.04/release/ubuntu-24.04-live-server-riscv64.img.gz">[ 下载 24.04 在线安装程序]</a>
```

Comment:
Can you remove the brackets ("[")? These are meant to be indicators of a `p-button` in the copy doc. Thanks